sos/report/plugins/iscsi.py

```diff
@@ -56,18 +56,19 @@ def postproc(self):
 
 
 class RedHatIscsi(Iscsi, RedHatPlugin):
+    """ RedHatPlugin's setup() will be invoked """
 
     packages = ('iscsi-initiator-utils',)
 
     def setup(self):
-        super(RedHatIscsi, self).setup()
+        """ RedHatPlugin's setup() will be invoked """
```

Comment:
Missing `super().setup()`, as well as in `DebianIscsi`?

Reply:
Since `setup` isn't doing anything, `super().setup()` isn't needed at all. So I'll remove this setup definition from both `RedHatIscsi` and `DebianIscsi` classes.